src/main/java/hudson/plugins/ansicolor/AnsiColorSimpleBuildWrapper.java

```diff
@@ -0,0 +1,158 @@
+package hudson.plugins.ansicolor;
+
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.console.ConsoleLogFilter;
+import hudson.console.LineTransformationOutputStream;
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.tasks.BuildWrapperDescriptor;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.Serializable;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import jenkins.tasks.SimpleBuildWrapper;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+public class AnsiColorSimpleBuildWrapper extends SimpleBuildWrapper implements Serializable {
```

Comment:
It suffices to just change the superclass of the existing `AnsiColorBuildWrapper`. (A `SimpleBuildWrapper` can be used by traditional projects too.)